src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, Countable {
+    private String color;
+    private String name;
+
+    public Figure() {
+    }
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public Figure(String color, String name) {
+        this.color = color;
+        this.name = name;
+    }
+
+    public void draw() {
+        System.out.println("Figure color: " + color + " name: " + name);
+    }
+
+    public double getArea() {

```

Comment:
The `getArea()` method should not be implemented here if it always returns 0. This method's implementation should be specific to each figure type and likely be abstract if `Figure` is intended to be a base class for other specific figure types.